app/models/host.rb

```diff
@@ -17,8 +20,13 @@ class Host < ApplicationRecord
   has_many :policies, through: :policy_hosts
   has_many :assigned_profiles, through: :policies, source: :profiles
 
-  validates :name, presence: true
-  validates :account, presence: true
+  def readonly?
+    true
+  end
+
+  def name
+    display_name
+  end
```

Comment:
Translation from what we called `name` to the view, which uses `display_name`. This change alone may actually solve the display_name issues we've had with the client.